odk2-src/conf.py

```diff
@@ -282,7 +282,7 @@ def setup(app):
 """
 odk_pdf = """
 
-_downloads/ODK2-Documentation.pdf
+_downloads/ODKX-Documentation.pdf
```

Comment:
First it should be "ODK-X".

Second, for the generated file to have that name you will need to change the build file so when it copies it into place it had the correct name.
See:
	mv OpenDataKit2.pdf ../_downloads/ODK2-Documentation.pdf

https://github.com/opendatakit/docs/blob/f77ead63c5272c14ebb767479ffdc1c2de0d8842/Makefile#L90